Description: do not call git when starting
Author: W. Martin Borgert <debacle@debian.org>
Origin: vendor
Last-Update: 2018-08-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/gajim/__init__.py
+++ b/gajim/__init__.py
@@ -1,17 +1,7 @@
 import os
-import subprocess
 
 __version__ = "1.0.99"
 
 IS_FLATPAK = False
 if os.path.exists('/app/share/run-as-flatpak'):
     IS_FLATPAK = True
-
-try:
-    p = subprocess.Popen('git rev-parse --short=12 HEAD', shell=True,
-                         stdout=subprocess.PIPE, stderr=subprocess.DEVNULL)
-    node = p.communicate()[0]
-    if node:
-        __version__ += '+' + node.decode('utf-8').strip()
-except Exception:
-    pass
